components/layout/DocsLayout.tsx

```diff
@@ -191,7 +191,7 @@ export default function DocsLayout({ post, navItems = {}, children }: IDocsLayou
                         </span>
                         <span className='font-sans text-sm text-gray-800 antialiased'>
                           Check&nbsp;
-                          <Link href='https://www.asyncapi.com/blog?tags=Release+Notes'>
+                          <Link href='https://www.asyncapi.com/blog?tags=Release+Notes' passHref legacyBehavior>
```

Comment:
Why don't we remove the `a` tag inside the Link and use the new Link Component only?

Reply:
I thought TailwindCSS but after carefully double-checking it's working fine. The latest commit should remove the `a` tag and put the behavior and style to the new `next/link` component